src/aggregator/aggregator/handler/config.go

```diff
@@ -209,6 +219,15 @@ func (c storagePolicyFilterConfiguration) NewConsumerServiceFilter() (services.S
 	return c.ServiceID.NewServiceID(), writer.NewStoragePolicyFilter(c.StoragePolicies)
 }
 
+type percentageFilterConfiguration struct {
+	ServiceID  services.ServiceIDConfiguration `yaml:"serviceID" validate:"nonzero"`
+	Percentage float64                         `yaml:"percentage" validate:"min=0.0,max=1.0"`
```

Comment:
that's a good point! it would be ideal to have dynamic config, however the config is one per consumer service id here, it's a little bit over complicated if we add kv with dict value type or add new kv item for every new service id. i will keep it as static config for now